instrumentation/opentelemetry-instrumentation-aws-lambda/src/opentelemetry/instrumentation/aws_lambda/__init__.py

```diff
@@ -262,9 +266,16 @@ def _instrument(self, **kwargs):
             self._wrapped_function_name,
         ) = lambda_handler.rsplit(".", 1)
 
+        flush_timeout = int(
```

Comment:
This can throw if the value cannot be converted to `int` so should be guarded against

Reply:
Good catch!